currency_rate_update_cmc/models/res_currency_rate_provider_CMC.py

```diff
@@ -0,0 +1,215 @@
+                    "SLEVIN", "DLISK", "CNNC", "VIP", "BXC", "PONZI", "ARB",
+                    "EGO", "KRONE", "ZMC", "KNC", "WBB", "ICON", "STARS",
+                    "BSC", "URC", "MILO", "DRM", "STEPS", "G3N", "IMPS",
+                    "RIDE", "BIOS", "XBTS", "BRAIN", "PIE", "PEX", "ORLY",
+                    "BOAT", "TAGR", "JOBS", "GEERT", "CWXT", "ACP", "DES",
+                    "RSGP", "PLNC", "ZNE", "CRT", "LIR", "OS76", "VOLT",
+                    "OFF", "CRDNC", "TOR", "ALTC", "SDP", "AGLC", "XOC",
+                    "XRC", "IBANK", "BIOB", "P7C", "COUPE", "ELS", "SOCC",
+                    "NODC", "CREVA", "MGM", "ULA", "SLFI", "CTIC2", "NANOX",
+                    "GSR", "LVPS", "CALC", "TYCHO", "DGCS", "TSTR", "PIZZA",
+                    "ABN", "EBT", "HT", "ATMC", "ELA", "OC", "WIC", "BCD",
+                    "CHAT", "W3C", "LBTC", "FRGC", "XTZ", "OCC", "BTCP",
+                    "OF", "XIN", "BSR", "BCX", "MLM", "SBTC", "BOS", "WC",
+                    "TER", "UBTC", "FIL", "$PAC", "LCC", "CLUB", "MSD", "ADK",
+                    "SWTC", "B2X", "MGC", "SBC", "NMS", "EAG", "SIC", "THS",
+                    "JIYO", "TMC", "IFC", "ENT", "TOK", "PCS", "GBG", "ACC",
+                    "BCA", "WA", "ANI", "XID", "JEW", "BT2", "VASH", "BIT",
+                    "TOKEN", "GMX", "GOD", "SPK", "VULC", "WOW", "CSC", "HC",
+                    "DAV", "ACES", "XRY", "SFE", "XQN", "EDRC", "ZBC", "FRN",
+                    "LEVO", "LEPEN", "SUP", "FONZ", "CYDER", "TCOIN", "CRYPT",
+                    "ROYAL", "DMC", "BAT", "SHA", "FLAP", "SLOTH", "WSX",
+                    "BIRDS", "BLAZR", "HIGH", "GRN", "GDC", "RBBT", "TOP",
+                    "RUNNERS", "ANTX", "MUSE", "UNRC", "ACN", "RICHX", "SKR",
+                    "PRN", "DON", "INDIA", "REGA", "XSTC", "KDC", "FID",
+                    "MARX", "XVE", "SJW", "LKC", "TURBO", "CMP", "TOPAZ",
+                    "APC", "GAIN", "ZSE", "LDCN", "NAMO", "PRIMU", "HNC",
+                    "NUMUS", "PAYP", "XTD", "BEST", "WINK", "AV", "SKULL",
+                    "BUB", "DCRE", "CME", "AKY", "UR", "GAY", "TELL", "HALLO",
+                    "FAZZ", "SIGMA", "SAK", "DUTCH", "SHELL", "MAGN", "OP",
+                    "HYPER", "BSN", "QBT", "CFC", "TODAY", "RUBIT", "POKE",
+                    "CHEAP", "DBG", "DASHS", "EGG", "OPES", "BET", "TEAM",
+                    "BAC", "NBIT", "MMXVI", "MONETA", "AXIOM", "MBL", "LTH",
+                    "FUTC", "BITOK", "BTBc", "SPORT", "CYC", "FRWC", "DUB",
+                    "GML", "CC", "VOYA", "QORA", "KARMA", "XAU", "TRICK",
+                    "IVZ", "X2", "PSY", "TCR", "DISK", "PRM", "HCC", "OMC",
+                    "LAZ", "RCN", "YES", "PDG", "KASHH", "RHFC", "TLE", "ASN",
+                    "UNC", "OCOW", "INF", "FRCT",
+                ]
+        return super()._get_supported_currencies(service)
+
+    allowed_base_currencies = [
```

Comment:
Quite generic name, may lead to collision with other modules

Reply:
Which other modules? This name was already in v11 of this module.

Reply:
Theoretical ones. Even taking into account that this variable name was first claimed by `currency_rate_update_cmc`, I'd still either rename it or would move it "upstairs" to the file scope.